framework/src/wifi_manager/wifi_utils.h

```diff
@@ -167,14 +167,14 @@ wifi_utils_result_e wifi_utils_get_info(wifi_utils_info *wifi_info);
 /**
  * @brief register wifi connection event callback (connect/disconnect event)
  *
- * @param[in]   connect_event_func    :  when wifi connect event received, function start
- * @param[in]   disconnect_event_func :  when wifi disconnect event received, function start
+ * @param[in]   linkup_event_func    :  when wifi linkup event received, function start
+ * @param[in]   linkdown_event_func :  when wifi linkdown event received, function start
  *
  * @return WIFI_UTILS_SUCCESS       :  success
  * @return WIFI_UTILS_FAIL          :  fail
  * @return WIFI_UTILS_INVALID_ARGS  :  input parameter invalid
  */
-wifi_utils_result_e wifi_utils_register_connection_callback(void *connect_event_func, void *disconnect_event_func);
+wifi_utils_result_e wifi_utils_register_callback(void *linkup_event_func, void *linkdown_event_func);
```

Comment:
The callbacks are relate to join, leave(SoftAP mode), connect, and disconnect(STA mode). so it's confusing to write linkup and down.

Reply:
We need to discuss more.